### DISCUSSION TITLE
Problems with formatting the code?

### DISCUSSION THREAD

**Discussion by koenigsdorfer** [kalkih/mini-graph-card#717] [Q&A] [answered]
G'day all,

this might not be the right spot for the question but I have trouble "formatting" the code correctly, in particular around "rect.bar:last-of-type "

I'm adding a new card to the lovelace dashboard as attached but the last bar is not filled red for some reason?

```
type: custom:mini-graph-card
entities:
  - entity: sensor.netatmo_netatmo_rain_gauge_rain_today
show:
  graph: bar
card_mod:
    style: |
    rect.bar:last-of-type 
    {
    fill: red;
    }
name: Rain
hours_to_show: 336
aggregate_func: max
group_by: date
```

![image](https://user-images.githubusercontent.com/87228221/152263165-6006f2ef-91b4-42be-92e9-40aef1ef320f.png)

**Comment by ildar170975** [comment, marked as answer]
https://community.home-assistant.io/t/lovelace-mini-graph-card/71385/1948?u=ildar_gabdullin

**Reply by jlsjonas** [reply]
from forum:
```
card_mod:
      style: |
        rect.bar:last-of-type {
           fill: red;
        }
```
It looks like you miss an indentation for the style lines @koenigsdorfer

**Reply by koenigsdorfer** [reply]
thanks mate, being new to this, I wasn't aware of the card mod to do css styling

**Reply by jlsjonas** [reply]
No problem, card_mod is indeed an external dependency, for specific _advanced_ css modifications
